adoc/Private-Registry-Installation.adoc

```diff
@@ -62,27 +68,27 @@ export KUBECONFIG=<full path to kubeconfig.yaml>
 The default configuration provided with the {spr} helm chart is not suited for production use!
 
 A separate YAML file (referred to as the `harbor-values.yaml` file in the following sections) needs to be populated with customized configuration values to be used during installation.
-The exact configuration options that can be customized, as well as the values that they can take are covered in detail in the next installation steps.
+The exact configuration options that can be customized, as well as the values that they can take, are covered in detail in the next installation steps.
 ====
 +
 [TIP]
 ====
-The full list of configuration options and default values that can overridden in the `harbor-values.yaml` file is included in the helm chart itself and can be viewed in YAML format by running the following command:
+The full list of configuration options and default values that can overridden in `harbor-values.yaml` is included in the helm chart itself, and can be viewed in YAML format by running the following command:
```

Comment:
Wouldn't that sentence need a plural? And it seems there is an additional typo. Just to make sure, can you please check: 

The full list of configuration options and default values that can [be?] overridden in `harbor-values.yaml` are included in the helm chart itself, ...

Reply:
Well spotted! Fixed.